synapse/config/repository.py

```diff
@@ -274,6 +279,9 @@ def default_config(self, data_dir_path, **kwargs):
         # synapse to issue arbitrary GET requests to your internal services,
         # causing serious security issues.
         #
+        # This must be specified if url_preview_enabled. It is recommended that you
+        # uncomment the following list as a starting point.
+        #
```

Comment:
Wouldn't it be sensible to add something like "0.0.0.0 and :: are always blacklisted" somewhere around here?

Reply:
probably.

Reply:
done